doc/Makefile

```diff
@@ -154,4 +154,4 @@ doctest:
 	      "results in $(BUILDDIR)/doctest/output.txt."
 s3:
 	aws s3 sync _build/html s3://www.glueviz.org/en/`git symbolic-ref -q --short HEAD`
```

Comment:
Thanks for spotting this, fixed this and the issue below in 197b862fa8ded05fa659d77f69b85f4ea305dc61